.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -15,56 +16,110 @@ body:
       placeholder: The bug is...
     validations:
       required: true
+
   - type: textarea
     id: reproduce
     attributes:
       label: Steps to Reproduce
-      description: Provide steps to reproduce this bug.
+      description: |
+        Provide steps to reproduce this bug.
+        If this bug is related to you application not functioning correctly with mirrord,
+        we would appreciate providing a link to repository with minimal reproducible example.
       placeholder: |
         1.
         2.
         3.
     validations:
       required: true
+
   - type: textarea
     id: debug
     attributes:
       label: Backtrace
       description: If your bug produces a backtrace, please include it here.
       render: shell
+
   - type: textarea
     id: layer-logs
     attributes:
       label: mirrord layer logs
-      description: If possible, reproduce the bug with the environment variable RUST_LOG set to 'trace', and attach the printed logs.
+      description: |
+        If possible, reproduce the bug and attach the relevant `mirrord-layer` logs.
+        You can enable them by setting `RUST_LOG` environment variable to `mirrord=trace`.
+        You will find them in your application's STDERR.
       render: shell
+
   - type: textarea
     id: intproxy-logs
     attributes:
-      label: Internal proxy logs
+      label: mirrord intproxy logs
+      description: |
+        If possible, reproduce the bug and attach the relevant `mirrord-intproxy` logs.
+        You can enable them by setting `internal_proxy.log_level` to `mirrord=trace` in your mirrord config.
```

Comment:
Is it possible to put links here? If so I'd link to the internal_proxy.log_level configuration docs (just so it's clear where in the JSON to put it)

Reply:
Yup, the link seems to render fine